src/components/Warning.js

```diff
@@ -1,24 +1,32 @@
 import React from "react"
 import styled from "styled-components"
-import Emoji from "./Emoji"
+import { Twemoji } from "react-emoji-render"
```

Comment:
FYI @ryancreatescopy - moving forward, we only want to import `Twemoji` in our `<Emoji />`component. Everything else should import our `<Emoji />`component.